activation/identities.go

```diff
@@ -18,21 +17,19 @@ func NewIdentityStore(db database.Database) *IdentityStore {
 	return &IdentityStore{db}
 }
 
-func getKey(key string) [32]byte {
-	return sha256.Sum256(util.Hex2Bytes(key))
+func getKey(key string) []byte {
```

Comment:
since the key can be arbitrary string, can this change cause the size of key become variable?

Reply:
we are using public key of the node here, but it also doesn't matter much if the length will be variable